### PR TITLE
Update PyTorch examples quantization API

### PR DESCRIPTION
**Pull request by changwangss**
Signed-off-by: changwa1 <chang1.wang@intel.com>

## Type of Change

API changed

## Description

enable examples for recommendation, speech recognition

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

**Review by PenghuiCheng — APPROVED** [review]

File: examples/.config/model_params_pytorch.json [hunk 1]
```diff
@@ -310,28 +301,19 @@
       "model_src_dir": "recommendation/dlrm/quantization/ptq/eager",
       "dataset_location": "/mnt/local_disk3/dataset/dlrm/dlrm/input",
       "input_model": "/mnt/local_disk3/dataset/dlrm/dlrm/dlrm_weight/tb00_40M.pt",
-      "yaml": "conf.yaml",
-      "strategy": "basic",
-      "batch_size": 16384,
-      "new_benchmark": false
+      "batch_size": 16384
     },
```
Comment (chensuyue) [inline examples/.config/model_params_pytorch.json:305]: require new parameter: “main_script”: “xxx.py”,

**Review by xin3he — APPROVED** [review]

File: examples/.config/model_params_pytorch.json (hunk 1 above)
Comment (chensuyue) [reply examples/.config/model_params_pytorch.json:305]: Because test code need to replace some config class, so need to know the name scripts.

**Comment by chensuyue** [comment]
1. example readme need to update. 
2. benchmark part need update into Config Class as well. 
3. run_benchmark.sh input parameter update. 
4. json param missing main_script.
Details pls check INC newAPI examples.pptx

**Comment by chensuyue** [comment]
Need to update performance mode with 'from neural_compressor import benchmark', use internal benchmark to run multi-instance.

**Pull request closed without merging**